include/zephyr/pm/device.h

```diff
@@ -576,7 +576,7 @@ int pm_device_power_domain_remove(const struct device *dev,
  *
  * @param dev Device instance.
  *
- * @retval true If device is currently powered
+ * @retval true If device is currently powered or not on a power domain
```

Comment:
In this case you should mention that returns true if pm is not enabled as well. What you think about adding the comment in the code here then ? 
```
If a device doesn't support PM or is not under a PM domain, it is assumed it is powered on.
```

Reply:
You mean as part of `@retval` or in the text block? Either way it makes sense since its more complete.

Reply:
I put it in the text block, please have a look